compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/DefaultArgumentStubGenerator.kt

```diff
@@ -180,6 +185,15 @@ open class DefaultArgumentStubGenerator(
         } else dispatchCall
     }
 
+    protected open fun needsSuperCallHandlerCheck(irFunction: IrSimpleFunction) = false
```

Comment:
I'd combine these two methods into one `generateSuperCallHandlerCheckIfNeeded` to avoid a stub implementation below

Reply:
Done!